test_queue.py

```diff
@@ -0,0 +1,77 @@
+# Test file for queue.py
+# Authors Mark, Efrain, Henry
+
+import pytest
+import queue as que
+
+def test_init():
+    """Test queue constructor."""
+    a = que.Queue()
+    assert a.front is None
+    assert a.back is None
+
```

Comment:
You have far too many assert statements per test - each test should only test ONE aspect of your code, and should ideally be 3 or less asserts statements. Your tests should help you pinpoint where your code breaks!